service/framework.go

```diff
@@ -391,8 +392,8 @@ func newPodFramework(config Config) (*framework.Framework, error) {
 				return nil, microerror.Mask(err)
 			}
 
-			meta.KeyVals["cluster"] = key.ClusterIDFromPod(pod)
-			meta.KeyVals["framework"] = "pod"
+			meta.KeyVals["object"] = path.Join(pod.Namespace, pod.Name)
+			meta.KeyVals["groupVersionKind"] = pod.GroupVersionKind().String()
```

Comment:
I like the version info. What is the value for pod actually?

Reply:
I think something like `core/v1, Kind=Pod`.